internal/core/item_rendering.rs

```diff
@@ -487,22 +526,80 @@ pub trait ItemRendererFeatures {
 /// After rendering an item, we cache the geometry and the transform it applies to
 /// children.
 #[derive(Clone)]
-pub struct CachedItemGeometryAndTransform {
-    /// The item's reported geometry, via [`ItemRc::geometry`].
-    pub geometry: LogicalRect,
-    /// The item's transform to apply to children, via [`ItemRc::children_transform`].
-    pub boxed_children_transform: Option<Box<ItemTransform>>,
-}
 
-impl From<(LogicalRect, Option<ItemTransform>)> for CachedItemGeometryAndTransform {
-    fn from((geometry, children_transform): (LogicalRect, Option<ItemTransform>)) -> Self {
-        Self { geometry, boxed_children_transform: children_transform.map(Into::into) }
-    }
+pub enum CachedItemGeometryAndTransform {
```

Comment:
For the name, this caches the bounding box rather than the geometry

Reply:
Good point :)